docs/application/dotnet/guides/uix/tts.md

```diff
@@ -378,9 +424,9 @@ void GetMode(Mode* mode)
 
 To operate TTS:
```

Comment:
Global Ref: https://developers.google.com/style/lists#introductory-sentences-for-lists
```suggestion
To operate TTS, follow these steps:
```